src/ServiceControl.Audit/Connection/ConnectionController.cs

```diff
@@ -0,0 +1,32 @@
+namespace ServiceControl.Audit.Connection
+{
+    using System.Web.Http;
+    using Infrastructure.Settings;
+    using Newtonsoft.Json;
+
+    public class ConnectionController : ApiController
+    {
+        readonly Settings settings;
+        readonly JsonSerializerSettings jsonSerializerSettings = new JsonSerializerSettings();
+
+        public ConnectionController(Settings settings) => this.settings = settings;
+
+        [Route("connection")]
+        [HttpGet]
+        public IHttpActionResult GetConnectionDetails() =>
+            Json(new
```

Comment:
Should we have a separate type for the result and approvals to ensure we detect any breaking changes?

Reply:
I added concrete types. There is an approval test for each transport in the multi-instance tests